_rules/id-value-unique-3ea0c8.md

```diff
@@ -152,14 +152,12 @@ The `xml:id` attribute is not considered by this rule.
 <div xml:id="my-div">This is my first element</div>
 ```
 
-
 #### Inapplicable Example 3
 
 These `id` attributes have an empty value.
 
 ```html
-<span id="">Hello</span>
-<span id="">world!</span>
+<span id="">Hello</span> <span id="">world!</span>
```

Comment:
Can you take out these prettier changes? This effects an example unrelated to the PR.

Reply:
This is actually somewhat difficult. Prettier is run automatically as a commit hook.
Pushing a non-prettied file actually requires going through a weird process to fore ignore it.

Reply:
The commit hook only goes through files that were changed in the commit.